app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -11,6 +11,24 @@
 )
 
 
+class NimbusStatusRestrictionMixin:
+    required_status = NimbusExperiment.Status.DRAFT
+
+    def validate(self, data):
+        data = super().validate(data)
+        if self.instance and self.instance.status != self.required_status:
+            status = self.instance.status
+            raise serializers.ValidationError(
+                {
+                    "experiment": [
+                        f"Nimbus Experiment has status '{status}', but can only "
+                        f"be changed when set to '{self.required_status}'."
+                    ]
+                }
+            )
+        return data
```

Comment:
Yeah I was gonna suggest we do this at the queryset lookup level rather than at the validation level, but this also gives a very nice clear error message so that's probably better than 'unable to mutate: no such experiment', so ya looks good 👍